test/test_maskedtensor.py

```diff
@@ -536,24 +692,154 @@ def test_masked_bmm(self):
             ]
         )
         attn_mask = attn_mask_bool.float().masked_fill_(attn_mask_bool, float("-inf"))
-        v = masked_bmm(x, x_mt.transpose(1, 2), attn_mask)
-        v.sum().backward()
 
-    def test_linear(self):
```

Comment:
What happened to this test?

Reply:
It wasn't actually testing an op on MaskedTensor afaik; it just performed `torch.nn.functional.linear` on a regular Tensor and then put it inside a `MaskedTensor`. Unless you had another test in mind to add?

Reply:
Does MaskedTensor support linear? If so we might need to add the test.